Filters/CDR0000339576.xml

```diff
@@ -109,6 +109,27 @@ OCECDR-3956: Allow CitationLinks in Image Captions
  </xsl:template>
 
 
+ <!--
+ ======================================================================
+ Display a ProtocolRef
+ The element should link to ClinicalTrials.gov using the following
+ format:
+    https://clinicaltrials.gov/show/NCT00004451
+ ====================================================================== -->
+ <xsl:template                   match = "ProtocolRef">
+  <a>
+   <xsl:attribute                 name = "href">
+    <!--
+    <xsl:text>https://clinicaltrials.gov/show/</xsl:text>
+    -->
+    <xsl:text>https://www.cancer.gov/clinicaltrials/</xsl:text>
+    <xsl:value-of               select = "@nct_id"/>
```

Comment:
... or the code?

Reply:
The comment was right for the first commit but then the requirements were modified and I forgot to update the comment.
I'll fix that.

Reply:
+1

You might want to remove the commented-out `<xsl:text/>` element while you're at it to eliminate any confusion about what it's doing there.